components/infieldprogresscircle/stories/infieldprogresscircle.stories.js

```diff
@@ -0,0 +1,121 @@
+import { Sizes } from "@spectrum-css/preview/decorators";
+import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { size } from "@spectrum-css/preview/types";
+import { default as ProgressCircle } from "@spectrum-css/progresscircle/stories/progresscircle.stories.js";
+import packageJson from "../package.json";
+import { InfieldProgressCircleGroup } from "./infieldprogresscircle.test.js";
+import { Template } from "./template.js";
+
+/**
+ * In-field progress circles are used in t-shirt size components such as [buttons](/docs/components-button--docs), [combo boxes](/docs/components-combobox--docs), and [pickers](/docs/components-picker--docs). The in-field progress circle can be used in place of an icon or in tight spaces when space is limited both vertically and horizontally.
+*/
+
+export default {
+	title: "In-field progress circle",
+	component: "InfieldProgressCircle",
+	argTypes: {
+		...ProgressCircle.argTypes,
+		size: size(["s", "m", "l", "xl"]),
+	},
+	args: {
+		...ProgressCircle.args,
+		rootClass: "spectrum-InfieldProgressCircle",
+	},
+	parameters: {
+		...ProgressCircle.parameters,
+		design: {
+			type: "figma",
+			url: "https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=14970-6050",
+		},
+		packageJson,
+	}
+};
+
+export const Default = InfieldProgressCircleGroup.bind({});
+Default.args = {};
+
+// ********* VRT ONLY ********* //
+export const WithForcedColors = InfieldProgressCircleGroup.bind({});
+WithForcedColors.args = Default.args;
+WithForcedColors.tags = ["!autodocs", "!dev"];
+WithForcedColors.parameters = {
+	chromatic: {
+		forcedColors: "active",
+		modes: disableDefaultModes,
+	},
+};
+
+// ********* DOCS ONLY ********* //
+
+export const Sizing = (args, context) => Sizes({
+	Template: Template,
```

Comment:
We can just write `Template,` here, as opposed to `Template: Template` if you want to reduce some repetitiveness.

Same thing for the Indeterminate story.